admin_app/src/components/ContentCard.tsx

```diff
@@ -32,20 +31,9 @@ export const ContentCard: React.FC<ContentCardProps> = (
       <div className="absolute flex top-2 right-2 ">
         {card.showEditButton ? (
           <>
-            <PencilIcon
```

Comment:
When using "readonly" user, you can still edit the card. It throws an error that the "Card could not be saved".
For readonly user we should open the window but: (a) not call it "Edit Content" but something else (b) Not show the Submit button (c) not let the user edit or resize the textarea

Reply:
Also double click to enlarge option is now gone.

Reply:
Also, I was thinking that we want "edit" to be more intentional and not happen by mistake if you just wanted to view it.

Reply:
Went back to earlier setup. The buttons are just farther away.